base/lnls-single-uid.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+#
+# Configure container filesystem so that single UID restriction is satisfied,
+# making ignore_chown_errors=True storage configuration work properly.
+
+echo "APT::Sandbox::User \"root\";" > /etc/apt/apt.conf.d/90-disable-sandbox.conf
```

Comment:
Use single quotes for the outer quotes, to avoid needing to escape the inner ones.

Reply:
Will do, thanks.